fearless/Common/View/UIFactory.swift

```diff
@@ -43,9 +43,21 @@ protocol UIFactoryProtocol {
         for delegate: AmountInputAccessoryViewDelegate?,
         locale: Locale
     ) -> UIToolbar
+
+    func createAccountView() -> DetailsTriangularedView
+
+    func createNetworkFeeView() -> NetworkFeeView
+
+    func createNetworkFeeConfirmView() -> NetworkFeeConfirmView
+
+    func createTitleValueView() -> TitleValueView
+
+    func createHintView() -> HintView
 }
 
 final class UIFactory: UIFactoryProtocol {
+    static let `default` = UIFactory()
```

Comment:
Instead of using reserved word I would suggest to rename it to defaultFactory

Reply:
But think about usage: ```UIFactory.default``` vs ```UIFactory.defaultFactory```